docs/docs/concepts/eventstore/overview.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Overview
+---
+
+ZITADEL is built on the [event sourcing pattern](../architecture), where changes are stored as events in an eventstore.
+
+## What is an eventstore?
+
+Traditionally, data is stored in relations as a state
+
+- A request needs to know the relations to select valid data
+- If a relation changes, the requests need to change as well
+- That is valid for actual, as well as for historical data
+
+An Eventstore on the other hand stores events, meaning every change that happens to any piece of data relates to an event.
+The data is stored as events in an append-only lof.
+
+- Think of it as a ledger that gets new entries over time, accumulative
+- To request data, all you have to do is to sum the events as the summary reflects the actual state
+- To investigate past changes to your system, you just select the events from your time range of interest
+- That makes audit/analytics very powerful, due to the historical data available to build queries
+
+## Benefits
+
+Audit: You have a built-in audit trail that tracks all changes over an unlimited period of time.
+Travel back in time: With our way of storing data we can show you all of your resources at a given point in time. 
+
+## Definitions
+
+Eventsourcing has some specific terms that are often used in our documentation. To unserstand how ZITADEL works it is important to understand this key definitions.
+
+### Events
+
+An event is something that happen in the system and got written to the database. This is the single source of truth.
+Events are immutable and the current state of your system is derived from the events.
+
+Possible Events:
+- user.added
+- user.changed
+- product.added
+- user.password.checked
+
+### Aggregate
```

Comment:
Are they stored in a write model?

Reply:
No an aggregate is actually just all events from a specific "object" together. The write model is used in the command side to write something to the eventstore.

Reply:
So I understand it is computed and stored in-memory and can't be inspected in the db.

Reply:
Maybe we can write that for clarity?